conjur/init_logic.py

```diff
@@ -0,0 +1,107 @@
+# Builtin
+import logging
+import sys
+import os.path
+from pathlib import Path
+
+# Third party
+import socket
+import yaml
+from OpenSSL import SSL
+
+class InitLogic:
+  """
+  InitLogic
+
+  This class holds the business logic for population the  conjurrc configuration details needed to connect with Conjur
+  """
+  DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
```

Comment:
`os.path.expanduser(os.path.join('~', '.conjurrc'))` is three times in the code . Put it in one place and use it from there

Reply:
moved to constants file